tensorflow/lite/experimental/litert/vendors/qualcomm/core/builders/quantize_op_builder.cc

```diff
@@ -35,4 +35,22 @@ std::vector<OpWrapper> BuildQuantizeOp(
   return res;
 }
 
+std::vector<OpWrapper> BuildDequantizeOp(
+    TensorPool& tensor_pool, const std::vector<TensorWrapperRef>& inputs,
+    const std::vector<TensorWrapperRef>& outputs) {
+  std::vector<OpWrapper> res;
+  const char* qnn_op = nullptr;
+  if (inputs[0].get().IsF16() && outputs[0].get().IsF32()) {
+    qnn_op = QNN_OP_CONVERT;
```

Comment:
We use cast in delegate.
```suggestion
    qnn_op = QNN_OP_CAST;
```

Reply:
Thank you for the review, I will work on submitting those Ops first.